katportalclient/client.py

```diff
@@ -1211,11 +1148,19 @@ def sensor_detail(self, sensor_name):
                         sensor_name,
                         [result['name'] for result in results[0:5]]))
         else:
-            raise tornado.gen.Return(results[0])
+            attrs = results[0].get('attributes')
+            result = {'name': results[0].get('name'),
+                      'description': attrs.get('description'),
+                      'params': attrs.get('params'),
+                      'katcp_name': attrs.get('katcp_name'),
+                      'units': attrs.get('units'),
+                      'type': attrs.get('type'),
+                      'component': results[0].get('component')}
+            raise tornado.gen.Return(result)
 
     @tornado.gen.coroutine
     def sensor_history(self, sensor_name, start_time_sec, end_time_sec,
-                       include_value_ts=False, timeout_sec=300):
+                       include_value_time=False, interval=0):
```

Comment:
This new `interval` parameter is not in the docstring.   I thought we were going to leave that work for a new PR?